exp/api/v1beta1/gcpmanagedcontrolplane_webhook_test.go

```diff
@@ -0,0 +1,252 @@
+/*
+Copyright 2022 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1beta1
+
+import (
+	"strings"
+	"testing"
+
+	. "github.com/onsi/gomega"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+var (
+	vV1_27_1       = "v1.27.1"
```

Comment:
This should be hardcoded? can we avoid that?

Reply:
This is simply used as a valid version placeholder but the value itself is not relevant. I'd say we won't need to update this in the foreseeable future, so it won't hurt that it is hard-coded. What do you think?